### PR TITLE
Set default Session Configuration's urlCache to nil

### PR DESCRIPTION
**Pull request by vahancouver**
[SDKS-2700](https://bugster.forgerock.org/jira/browse/SDKS-2700)
[iOS] Inadequate Cache Control - Mobile Application

Approach 2

### CODE REVIEW

File: FRCore/FRCore/Network/RestClient.swift [hunk 1]
```diff
@@ -33,6 +33,7 @@ public class RestClient: NSObject {
             config.httpCookieStorage = nil
             config.httpCookieAcceptPolicy = .never
             config.httpShouldSetCookies = false
+            config.urlCache = nil
```
Comment (george-bafaloukas-forgerock) [inline FRCore/FRCore/Network/RestClient.swift:36]: Maybe let's add some comments here on why we are doing this, same for the cookieStorage

**Review by george-bafaloukas-forgerock — APPROVED** [review]

File: FRCore/FRCore/Network/RestClient.swift (hunk 1 above)
Comment (vahancouver) [reply FRCore/FRCore/Network/RestClient.swift:38]: Added comments

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Review by spetrov — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**